apps/condo/domains/analytics/utils/helpers.spec.ts

```diff
@@ -0,0 +1,335 @@
+import dayjs, { Dayjs } from 'dayjs'
+import crypto from 'crypto'
+import { TicketAnalyticsGroupBy, TicketGroupedCounter } from '@app/condo/schema'
+import {
+    getAggregatedData,
+    filterToQuery,
+    ticketAnalyticsPageFilters,
+    getChartOptions,
+    isEmptyAnalyticsData,
+} from './helpers'
+
+describe('Helpers', () => {
+    describe('queryUtils', () => {
+        describe('filterToQuery',  () => {
+            describe('it should correctly generate query if', () => {
+                const property = {
+                    id: crypto.randomBytes(64).toString('hex'),
+                    value: 'property address',
+                }
+                const selectedRange: [Dayjs, Dayjs] = [dayjs().subtract(1, 'week').startOf('day'), dayjs().endOf('day')]
+
+                it('filter contains property', () => {
+                    const filter: ticketAnalyticsPageFilters = {
+                        range: selectedRange,
+                        addressList: [property],
+                        specification: 'day',
+                        responsibleList: [],
+                        executorList: [],
+                        classifierList: [],
```

Comment:
It's unclear how these properties are participating in execution, because they are missing in `expectedResult`